getnacelle/nacelle-js
# feat: Storefront SDK `2.0.0` GA

## Why are these changes introduced?

Addresses [ENG-8178](https://nacelle.atlassian.net/browse/ENG-8178) 

> Version 2.0.0 of @nacelle/storefront-sdk

## What is this pull request doing?

This PR is the culmination of months of work on a next major version of [`@nacelle/storefront-sdk`](https://www.npmjs.com/package/@nacelle/storefront-sdk).

For information about the improvements and new features associated with Storefront SDK 2.x, please refer to [docs.nacelle.com/docs/storefront-sdk-major-version-2](https://docs.nacelle.com/docs/storefront-sdk-major-version-2).

## How to Test

1. Check out the `ENG-8179/storefront-sdk-2.0` branch
2. `npm run bootstrap` as needed
3. Navigate to `packages/storefront-sdk`
4. `npm run build` - the package should build without errors
5. `npm run coverage` - all tests should pass with 100% coverage
6. Navigate to `packages/storefront-sdk-plugins/commerce-queries`
7. `npm run build` - the package should build without errors
8. `npm run coverage` - all tests should pass with 100% coverage

## Checklist



[ENG-8178]: https://nacelle.atlassian.net/browse/ENG-8178?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
